<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>今天你的安排是：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_study">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>学习</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_game">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打游戏</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_work">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>工作</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
